### PR TITLE
[T3A5][F12-A2]Lee Jun Han Bryan

### PR DESCRIPTION
**Pull request by bryanleejh**
Ready for review

### CODE REVIEW

**Comment by bryanleejh** [comment]
Some issues at line 100 of TextUI.java file. Problems with showToUser(). Suspect because haven't shifted formatting function in showResultToUser over to Formatter.java

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import static seedu.addressbook.common.Messages.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.addressbook.Main;
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class Formatter {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/ui/Formatter.java]: Remember to add class header comments.

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -75,45 +56,36 @@ private boolean isCommentLine(String rawInputLine) {
...
-                version,
-                MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
-                storageFileInfo,
-                DIVIDER);
+        showToUser(Formatter.welcomeMessageFormat(storageFileInfo));
     }
 
     public void showGoodbyeMessage() {
-        showToUser(MESSAGE_GOODBYE, DIVIDER, DIVIDER);
+    	showToUser(Formatter.goodByeMessageFormat());
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/ui/TextUi.java]: Indentation here seems off.
Seems to be a tab as compared to space.

**Review by brandonyeoxg — COMMENTED** [review]
@bryanleejh  Some comments added. Please close the PR after reading comments.

Try your best to solve the issue on google/stackoverflow.

File: src/seedu/addressbook/ui/Formatter.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	}
+	
+	/** Returns format of message asking for user command. */
+	public static String echoUserCommandFormat(String fullInputLine) {
+		return LINE_PREFIX + "[Command entered:" + fullInputLine + "]" + LS;
+	}
+	
+	/** Formats welcome message to user */
+	public static String welcomeMessageFormat(String storageFileInfo) {
+		return showToUserFormat(DIVIDER, DIVIDER, MESSAGE_WELCOME, Main.VERSION, MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/ui/Formatter.java]: is this line too long?
Check the coding standard to know the max chars per line.

Comment (bryanleejh) [reply src/seedu/addressbook/ui/Formatter.java:53]: 120 characters!

File: src/seedu/addressbook/ui/Formatter.java (hunk 1 above)
Comment (bryanleejh) [reply src/seedu/addressbook/ui/Formatter.java:12]: okay (:

**Pull request closed without merging**